pkg/networkmanager/dialogs-common.jsx

```diff
@@ -224,8 +244,13 @@
             dlg = <IpSettingsDialog topic="ipv4" {...properties} />;
         else if (type == 'ipv6')
             dlg = <IpSettingsDialog topic="ipv6" {...properties} />;
+
         if (dlg)
-            Dialogs.show(dlg);
+            resolveDeps(type).then(() => {
+                Dialogs.show(dlg);
+            }).catch((e) => {
+                console.error(e);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19024-20230719-101929-332f4bf1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)